### PR TITLE
ci: enable godot linter

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Long Dai <long0dai@foxmail.com>

# Description

Enable godot linter.

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

**Comment by artursouza** [comment]
I am fine with that. But build needs to pass.

**Comment by daixiang0** [comment]
@artursouza please check it, codecov seems not correctly.

File: pkg/sentry/certs/store.go [hunk 1]
```diff
@@ -80,7 +80,7 @@ func CredentialsExist(conf config.SentryConfig) (bool, error) {
 	return false, nil
 }
 
-/* #nosec */
+/* #nosec. */
```
Comment (artursouza) [inline pkg/sentry/certs/store.go:83]: Can this be handled as an exception? This is odd.

**Review by artursouza — CHANGES_REQUESTED** [review]
/cc @yaron2 do you agree to this change in our code style?

File: pkg/sentry/certs/store.go (hunk 1 above)
Comment (daixiang0) [reply pkg/sentry/certs/store.go:83]: Sure.

**Review by yaron2 — APPROVED** [review]

**Comment by yaron2** [comment]
> /cc @yaron2 do you agree to this change in our code style?

Yep

**Review by artursouza — APPROVED** [review]

**Pull request merged**